### PR TITLE
Remove duplicated hide colum option

### PR DESCRIPTION
**Pull request by LouisDelbosc**
Related issue #290 

The issue has not been checked, maybe there is a reason why you have two same option. Nonetheless, here the fix.

Tell me if you have any comments.

Screenshot 2022-09-27 at 15 06 15

### CODE REVIEW

**Comment by berhalak** [comment]
Hi @LouisDelbosc,

Actually, can you remove this option from `ColumnContextMenu` and leave that one in `MultiColumnMenu` (also, the logic for disabling it is missing check if the menu is on `Raw Data View`).
`MultiColumnMenu` is part of `ColumnContextMenu`, so with your change, it would be removed from the menu that is shown when multiple columns are selected.

**Comment by LouisDelbosc** [comment]
@berhalak thanks for you comment.

Is it more robust like this ?

File: app/client/ui/GridViewMenus.ts [hunk 1]
```diff
@@ -58,7 +58,7 @@ export function calcFieldsCondition(fields: ViewFieldRec[], condition: (f: ViewF
 }
 
 export function ColumnContextMenu(options: IColumnContextMenu) {
-  const { disableModify, filterOpenFunc, colId, sortSpec, isReadonly, isRaw } = options;
+  const { disableModify, filterOpenFunc, colId, sortSpec, isReadonly } = options;
```
Comment (berhalak) [inline app/client/ui/GridViewMenus.ts:61]: Looks very good. The only thing is this logic with `isRaw`. `Hide columns` in MultiColumnMenu should be hidden when `isRaw` is set to true. Can you add this logic there?

**Review by berhalak — CHANGES_REQUESTED** [review]

**Review by berhalak — APPROVED** [review]
Thanks, works great!

**Pull request merged**